src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.scss

```diff
@@ -6,3 +6,35 @@
 .selected-action {
   font-weight: lighter;
 }
+
+.wrapper {
+  display: inline-flex;
+  width: 100%;
+  padding: 20px 10px 20px 20px;
```

Comment:
We have some CSS classs to use for getting margins and paddings (they were introduced at a later staged so unfortunately there is still places that gets their margin/padding from other CSS classes).

Instead of setting the padding in here, you should use the classes `m-1, mt-2, p-3, pb-4` etc.
So in your HTML it would look something like this:
```
<div class="p-3 pt-2">
```

Reply:
Code standard document updated with this information.